### ISSUE TITLE
SSH keys should accept lp:name and gh:name 

### ISSUE THREAD

**Issue by bac** [juju/juju-gui#2551]
juju supports a means to import an ssh key from Launchpad or Github.

```
$ juju import-ssh-key -h
Usage: juju import-ssh-key [options] <lp|gh>:<user identity> ...

Summary:
Adds a public SSH key from a trusted identity source to a model.
```

It would be mighty handy if the SSH entry on the cloud credentials page allowed the same. Would be way more friendly than pasting in a fingerprint.

**Comment by frankban** [comment]
The launchpad API for getting SSH keys is `https://api.launchpad.net/1.0/~<USER>/sshkeys`

**Comment by mitechie** [comment]
+1 to auto loading ssh keys and I think we can even try to invisibly load them during account creation as part of getting SSO data. We know their username then and so we seem like a nice seamless system (all canonical single tool) if we were to load them then and have them waiting for you at the deployment flow. 

Then we'll still need some mechanism in the profile page to trigger updates/changes that a user might make to those ssh keys over time. (load new ones that weren't there when the account was created, add ones directly to jaas for users that aren't using launchpad, etc)

**Comment by frankban** [comment]
The basic support for importing github and launchpad ssh keys landed in Morn, and will be included in the next release (2.9.0).
